integration_tests/features/support/steps.js

```diff
@@ -268,26 +275,26 @@ When(/I send (.*) tari from (.*) to (.*),(.*) at fee (.*)/, async function (tari
 When(/I wait (.*) seconds/, {timeout: 600*1000}, async  function (int) {
     console.log("Waiting for", int, "seconds");
     await sleep(int*1000);
+    console.log("Waiting finished");
 });
 
 Then(/Batch transfer of (.*) transactions was a success from (.*) to (.*),(.*)/,  async function (txCount,walletA,walletB,walletC) {
-   var WalletA = this.getWallet(walletA);
-   var ClientA = WalletA.getClient();
-   var WalletB = this.getWallet(walletB);
-   var ClientB = WalletB.getClient();
-   var WalletC = this.getWallet(walletC);
-   var ClientC = WalletC.getClient();
+   let WalletA = this.getWallet(walletA);
```

Comment:
JS vars should be camelcase

```suggestion
   let walletA = this.getWallet(walletA);
```

Reply:
Think I got all of them